packages/samples/atomic-react/src/App.tsx

```diff
@@ -1,218 +1,40 @@
-            />
-
-            <AtomicQueryError />
-            <AtomicNoResults />
-          </AtomicLayoutSection>
-          <AtomicLayoutSection section="pagination">
-            <AtomicLoadMoreResults />
-          </AtomicLayoutSection>
-        </AtomicLayoutSection>
-      </AtomicSearchLayout>
-    </AtomicSearchInterface>
-  );
-};
-
-function MyTemplate(result: Result) {
+  const [page, setPage] = useState(LIST_PAGE);
   return (
     <>
-      <AtomicResultSectionBadges>
-        <AtomicResultBadge field="ec_brand" />
-      </AtomicResultSectionBadges>
-      <AtomicResultSectionVisual>
-        <AtomicResultImage field="ec_images" />
-      </AtomicResultSectionVisual>
-      <AtomicResultSectionTitle>
-        <AtomicResultLink />
-      </AtomicResultSectionTitle>
-      <AtomicResultSectionTitleMetadata>
-        <AtomicResultRating field="ec_rating" />
-        <AtomicResultPrintableUri maxNumberOfParts={3} />
-      </AtomicResultSectionTitleMetadata>
-      <AtomicResultSectionEmphasized>
-        <AtomicResultNumber field="ec_price">
-          <AtomicFormatCurrency currency="USD" />
-        </AtomicResultNumber>
-      </AtomicResultSectionEmphasized>
-      <AtomicResultSectionExcerpt>
-        <AtomicResultText field="ec_shortdesc" />
-      </AtomicResultSectionExcerpt>
-      <AtomicResultSectionBottomMetadata>
-        <AtomicResultFieldsList>
-          {result.raw.cat_platform !== undefined && (
-            <>
-              <span className="field-label">
-                <AtomicText value="Platform" />
-              </span>
-              <AtomicResultText field="cat_platform" />
-            </>
-          )}
-          {result.raw.cat_condition !== undefined && (
-            <>
-              <span className="field-label">
-                <AtomicText value="Condition" />
-              </span>
-              <AtomicResultText field="cat_condition" />
-            </>
-          )}
-          {result.raw.cat_categories !== undefined && (
-            <>
-              <span className="field-label">
-                <AtomicText value="Tags" />
-              </span>
-              <AtomicResultMultiValueText field="cat_categories" />
-            </>
-          )}
-        </AtomicResultFieldsList>
-      </AtomicResultSectionBottomMetadata>
+      <header>
+        <span className="pageTitle">{page} example</span>
+        <ul>
+          <HeaderLink page={LIST_PAGE} currentPage={page} setPage={setPage} />
+          <HeaderLink
+            page={FOLDED_LIST_PAGE}
+            currentPage={page}
+            setPage={setPage}
+          />
+          <HeaderLink
+            page={FOLDED_CUSTOM_CHILDREN_PAGE}
+            currentPage={page}
+            setPage={setPage}
+          />
+        </ul>
+      </header>
+      {page === LIST_PAGE && <List />}
+      {page === FOLDED_LIST_PAGE && <Folded />}
+      {page === FOLDED_CUSTOM_CHILDREN_PAGE && <CustomChildren />}
     </>
```

Comment:
When just looking at the file names, I find it difficult to figure out which components are pages.

I think it would be useful to rename them so their names contain `Page` (e.g., `ResultListPage`, `FoldedResultListPage`, `CustomFoldedResultListPage`).

Additionally, I think it would be useful to nest all three components in a `src/pages` folder.

Reply:
you're right! I didn't give it too much thought because I thought it was more for us to test, but I guess external people might use it too.. and also we should keep it tidy anyway! will follow your suggestions